testsuite/run_sets/build_validation/build_validation_retail_proxy.yml

```diff
@@ -1,5 +1,10 @@
 # This file describes the order of features in a Build Validation test suite run.
 
+## Build Kiwi images
+## (expected time of building is around 20 minutes per image)
+- features/build_validation/retail/sle12sp5_buildhost_build_kiwi_image.feature
+- features/build_validation/retail/sle15sp4_buildhost_build_kiwi_image.feature
+
```

Comment:
In the future, we might want to have a separated yaml file for the building images stage.
I think it merits its own stage because it has a responsibility very well defined and separated from retail. And also, this is a strong dependency for the next features, so it's better that if one of these features fail, we don't run the features currently below these ones.

Reply:
Beware of the risk to run them after the proxy if you do that. If we do that, we should explicit the order in the comments.

Also, would be a bit less convenient to use with rake.

Other than that, no strong feelings.

Reply:
Yes. That's what I mean, have a new run_set `build_validation_retail_build_images.yml` and run them through Rake, in a new Jenkins pipeline stage, just before `build_validation_retail_proxy.yml`